module-4/backend/week_one.md

```diff
@@ -1,19 +1,53 @@
 ## Week One - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What's the most useful thing you learned from completing the intermission week work?
+
+  Learning how to write tests. I had some prior exposure to JavaScript but I had not done anything with testing.
+
 2. What are some tools to help debug JavaScript code?
+
+  Pry, browser console, `console.log()`
+
 3. What are some tools you need in order to unit test your JavaScript?
+
+  Mocha and Chai
+
 4. What is the syntax for invoking a function? Give an example.
+
+  `foo();`
+
 5. What's `this` in JavaScript?
+
+  `this` is a scoped instance of what it is called in.
+
 6. What is Webpack and why is it useful?
+
+  Webpack compiles our code into one file so that we don't need to link multiple script files to a page.
+
 7. When/why do you want to use event delegation?
+
+  When we want one event listener to fire off multiple different functions. Prevents us from writing more code than we need.
```

Comment:
Not quite - this is more for attaching event listeners to things that may have not been added to the DOM yet (items on a to-do list, for example)